components/patient-data-sharing/push-ui/src/main/resources/PhenoTips/PushPatient.xml

```diff
@@ -773,6 +775,8 @@
         var groupSelection = this._remoteGroupLabel.replace("__serverID__", this._selectedServer);
         $$('#group-selection h2')[0].innerHTML = groupSelection;
 
+        if (serverData.registrationURL != "") {$('user-selection').insert(new Element('p', {'class' : 'intro xHint'}).insert("$!escapetool.javascript($services.localization.render('phenotips.PushPatient.remoteRegistration'))".replace("__serverID__", this._selectedServer).replace("__registrationUrl__", "&lt;a target=\"_blank\" href=\""+serverData.registrationURL+"\"&gt;"+serverData.registrationURL+"&lt;/a&gt;")));}
```

Comment:
This appears to keep adding new elements for each server you attempt to push to.

To reproduce the issue:
- add 2 more push targets, one with a registration url and one without
- create a patient
- click on Push to PhenomeCentral: all looks fine
- close the dialog and click on the push target WITHOUT registration URL: the message for signing up to PhenomeCentral is still displayed
- close the dialog and click on the other push target WITH registration URL: both the message for signing up to PhenomeCentral and the message for signing up to this push target are displayed